app/org/sagebionetworks/bridge/play/interceptors/ExceptionInterceptor.java

```diff
@@ -60,8 +68,12 @@ private Result getResult(final Throwable throwable) throws JsonProcessingExcepti
             return Results.status(cre.getStatusCode(), info.toString());
         }
         String message = getMessage(throwable, status);
-        final ExceptionMessage exceptionMessage = new ExceptionMessage(throwable, message);
-        return Results.status(status, Json.toJson(exceptionMessage));
+        
+        ObjectNode node = (ObjectNode)BridgeObjectMapper.get().valueToTree(throwable);
+        node.put("message", message);
+        node.remove(UNEXPOSED_FIELD_NAMES);
+        
+        return Results.status(status, node);
```

Comment:
Should this filter out things like healthCode, studyId, etc? Or is that the responsibility of the code throwing the exception?